articalpage.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+$id = $_GET['blog'];
+
+require('includes/getBlogPosts.php');
+
+$blogpost = $blogposts[$id];
+?>
+
+
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="css/navbar.css">
+    <link rel="stylesheet" type="text/css" href="css/header.css">
+    <link rel="stylesheet" type="text/css" href="css/mainbody.css">
+    <link rel="stylesheet" type="text/css" href="css/footer.css">
+    <link rel="stylesheet" type="text/css" href="css/global.css">
+
+    <title> Luke Winsley </title>
+</head>
+
+<body>
+<div id="wrapper">
+    <div id="navbar">
+        <div id="logo"> <img src="images/logo.png" alt="logo"> </div>
+        <div class="navbutton"> <a href="index.php"> Home </a> </div>
+        <div class="navbutton"> <a href="aboutme.html"> About Me </a> </div>
+        <div class="navbutton"> <a href="portfoliopage.html"> Portfolio </a> </div>
+        <div class="navbutton"> <a href="contactpage.html"> Contact </a> </div>
+        <div class="navbutton"> <a href="bloglistpage.php"> Blog </a> </div>
+    </div>
+
+    <div id="header">
+        <?php
+        echo "<h1>" . $blogpost->title . "</h1>";
+        ?>
+    </div>
+
+    <div id="articalcontainer">
```

Comment:
"Article" is mis-spelled throughout - anybody looking at your code may think that if your code is wrong, any content you produce/upload may be wrong or not spellchecked. Command-r in php storm should find and replace all instances of mis-spelling. BUT this may cause problems - you might want to refactor instead - this should keep any connections.
Picky, but possibly important to somebody looking to employ you!!